lineage/Analyze.py

```diff
@@ -91,68 +91,32 @@ def Results(tHMMobj, pred_states_by_lineage: list, LL: float) -> dict[str, Any]:
     # Instantiating a dictionary to hold the various metrics of accuracy and scoring for the results of our method
     results_dict: dict[str, Any]
     results_dict = {}
+    tHMMobj, pred_states = permute_states(tHMMobj, pred_states_by_lineage)
```

Comment:
Here again, does Results() need `pred_states_by_lineage`, or can it just retrieve this my `tHMM.predict()`?

Reply:
This needs editing a couple of things, specifically, the output of `Analyze()` is linked to this, is it alright if we leave it this way? 
I changed the `permute_states()` and it only takes in the `tHMMobj`.